src/lib/modules/server/terminal/opencode-watcher.ts

```
@@ -0,0 +1,600 @@
+		id?: string;
+		role?: string;
+		content?: unknown;
+		stop_reason?: string;
+	};
+}
+
+// ── Per-session Watcher State ────────────────────────────────────────
+
+interface WatchState {
+	sessionId: string;
+	intervalHandle: ReturnType<typeof setInterval>;
```

ESLint

[error] Expected "intervalHandle" to come before "sessionId".

(perfectionist/sort-interfaces)